### ISSUE TITLE
Possible to cast to Google Assistant enabled device?

### ISSUE THREAD

**Issue by dolanmiu** [googlesamples/assistant-sdk-python#161]
Is it possible to cast **to** a Google Assistant enabled device?

For example, I install this on a Raspberry Pi, then on my computer, I cast the audio to the Pi.

Like what Google Home can do

**Comment by Fleker** [comment]
No, casting is a slightly separate component from the Assistant SDK.
